booster_bdd/features/src/stackAnalyses.py

```diff
@@ -25,33 +26,57 @@ def getReportKey (self, codebaseUrl):
                    'X-Git-Provider': 'GitHub',
                    'Content-Type': 'application/x-www-form-urlencoded'}
 
-        # print ('the target URL is:' + codebaseUrl)
+        print ('the target URL is:' + codebaseUrl)
 
         # Remove the ".git" suffix from the URL
         shortenedUrl = codebaseUrl
 
         if shortenedUrl.endswith(".git"):
             shortenedUrl = codebaseUrl[:-4]
-        # print ('the shortened URL is:' + shortenedUrl)
+        print ('the shortened URL is: {}'.format(shortenedUrl))
+
+        spaceName = helpers.getSpaceName()
+        projectName = os.getenv("PROJECT_NAME")
+        gitRepo = spaceName + "-" + projectName
+        print ('the gitRepo name is: {}'.format(gitRepo))
+
+        cloneUrl = codebaseUrl.replace('https://github.com/', 'git@github.com:')
+        print ('the cloneUrl = {}'.format(cloneUrl))
+
+        # Invoke shell script to create and upload to master effective pom
+        try:
+            testOutput = subprocess.check_output(['./test.sh', cloneUrl, gitRepo])
+            print ('Output = {}'.format(testOutput))
+        except Exception as e:
+            print('Unexpected subprocess exception: {}'.format(e))
+
+        print ('the shortened URL is: {}'.format(shortenedUrl))
 
         # TODO - The call is failing to all repos other than booster repos
         #shortenedUrl = 'https://github.com/wildfly-swarm-openshiftio-boosters/wfswarm-rest-http'
 
+        #payload = {
+        #    'github_url': shortenedUrl,
+        #    'source': 'osio',
+        #    'github_ref': 13
+        #}
+        filePathStr = './tempDir/' + gitRepo
+        print ('the pom.xml filePath is: {}'.format(filePathStr))
         payload = {
-            'github_url': shortenedUrl,
-            'source': 'osio',
-            'github_ref': 13
+            'filePath': filePathStr,
```

Comment:
@ldimaggi - 
use `filePath[]` and `manifest[]` as a payload. Its actually an array list. It should work.